src/legacy/core_plugins/kibana/public/discover/components/field_chooser/discover_field_search.tsx

```diff
@@ -68,7 +68,7 @@ export function DiscoverFieldSearch({
         defaultMessage: 'Show field filter settings',
       });
   const searchPlaceholder = i18n.translate('kbn.discover.fieldChooser.searchPlaceHolder', {
-    defaultMessage: 'Search fields',
+    defaultMessage: 'Search for fields',
```

Comment:
I prefer the previous, for what little it's worth.

Reply:
I talked to @wylieconlon, and he suggested `Search field names`.